client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPlugin.js

```diff
@@ -8,760 +8,129 @@
       deployment,
-    };
-  }
-
-  async getDefaultEndpoint(tab) {
-    const credentials = {
-      basicAuthUsername: '',
-      basicAuthPassword: '',
-      clientId: '',
-      clientSecret: '',
-      camundaCloudClientId: '',
-      camundaCloudClientSecret: ''
-    };
-
-    let endpoint = {
-      id: generateId(),
-      targetType: CAMUNDA_CLOUD,
-      authType: AUTH_TYPES.NONE,
-      contactPoint: '',
-      oauthURL: '',
-      audience: '',
-      camundaCloudClusterUrl: '',
-      ...credentials,
-      rememberCredentials: false
-    };
-
-    const previousEndpoints = await this.getEndpoints();
-    if (previousEndpoints.length) {
-      endpoint = {
-        ...credentials,
-        ...previousEndpoints[0]
-      };
-    }
-
-    // #2375 => if we only have a clusterId, transform it to clusterURL for backwards-compatability
-    if (!endpoint.camundaCloudClusterUrl && endpoint.camundaCloudClusterId) {
-      endpoint.camundaCloudClusterUrl = createCamundaCloudClusterUrl(endpoint.camundaCloudClusterId);
-    }
-
-    return endpoint;
-  }
-
-  async getDefaultConfig(savedConfig, tab) {
-    const deployment = {
-      name: withoutExtension(tab.name)
-    };
-
-    const endpoint = await this.getDefaultEndpoint(tab);
-
-    return {
-      deployment: {
-        ...deployment,
-        ...savedConfig.deployment
-      },
-      endpoint: {
-        ...endpoint,
-        ...savedConfig.endpoint
-      }
-    };
-  }
-
-  async saveEndpoint(endpoint) {
-    const existingEndpoints = await this.getEndpoints();
-
-    const updatedEndpoints = addOrUpdateById(existingEndpoints, endpoint);
-
-    await this.setEndpoints(updatedEndpoints);
-
-    return endpoint;
-  }
-
-  getEndpoints() {
-    return this.props.config.get(ZEEBE_ENDPOINTS_CONFIG_KEY, []);
-  }
-
-  setEndpoints(endpoints) {
-    return this.props.config.set(ZEEBE_ENDPOINTS_CONFIG_KEY, endpoints);
-  }
-
-  getTabConfiguration(tab) {
-    return this.props.config.getForFile(tab.file, DEPLOYMENT_CONFIG_KEY);
-  }
-
-  setTabConfiguration(tab, configuration) {
-    return this.props.config.setForFile(tab.file, DEPLOYMENT_CONFIG_KEY, configuration);
-  }
-
-  onDeploymentSuccess(response, configuration, options = {}) {
-    const { config, savedTab } = configuration;
-
-    const {
-      displayNotification,
-      triggerAction
-    } = this.props;
-
-    const {
-      endpoint
-    } = config;
-
-    const {
-      isStart
-    } = options;
-
-    const {
-      gatewayVersion
-    } = options;
-
-    const content = endpoint.targetType === CAMUNDA_CLOUD ?
-      <CloudLink endpoint={ endpoint } response={ response } />
-      : null;
-
-    if (!options.skipNotificationOnSuccess) {
-      displayNotification({
-        type: 'success',
-        title: this._getSuccessNotification(savedTab),
-        content: content,
-        duration: 8000
-      });
-    }
-
-    // notify interested parties
-    triggerAction('emit-event', {
-      type: 'deployment.done',
```

Comment:
It looks that this was removed without replacement, correct? @philippfromme 
[I cannot find `'deployment.done'` in the zeebe-plugin anymore](https://github.com/search?q=repo%3Acamunda%2Fcamunda-modeler%20%27deployment.done%27&type=code).